doc/netplan-yaml.md

```diff
@@ -591,7 +591,20 @@ Match devices by MAC when setting options like: `wakeonlan` or `*-offload`.
 - **neigh-suppress** (scalar) – since **0.105**
 
   > Takes a boolean. Configures whether ARP and ND neighbor suppression is
-  > enabled for this port. When unset, the kernel's default will be used.
+  > enabled for this bridge port. When unset, the kernel's default will be
+  > used.
+
+- **hairpin** (scalar) – since **0.108**
+
+  > Takes a boolean. Configures whether traffic may be sent back out of the
+  > bridge port on which it was received. When this flag is false, then the
+  > bridge will not forward traffic back out of the receiving port. When
+  > unset, the kernel's default will be used.
```

Comment:
todo: "When unset, the **backend's** default will be used."

(This is FALSE for NetworkManager, and the kernel's value for systemd-netword)